<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>477</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="settingsButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/login_setting.png</normaloff>:/image/login_setting.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="minButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/login_min.png</normaloff>:/image/login_min.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="closeButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/login_close.png</normaloff>:/image/login_close.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageLogin">
      <widget class="QLabel" name="loginTitle">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用户登陆</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginUser">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>110</y>
         <width>244</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>测试</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="loginCheckBoxPwd">
       <property name="geometry">
        <rect>
         <x>149</x>
         <y>234</y>
         <width>105</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
        </font>
       </property>
       <property name="text">
        <string>记住密码</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginPwd">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>170</y>
         <width>244</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>123456</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>129</x>
         <y>109</y>
         <width>112</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
      <widget class="QToolButton" name="loginRegButton">
       <property name="geometry">
        <rect>
         <x>311</x>
         <y>233</y>
         <width>187</width>
         <height>24</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>没有账号? 马上注册</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>129</x>
         <y>169</y>
         <width>109</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>密     码：</string>
       </property>
      </widget>
      <widget class="QToolButton" name="loginSureButton">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>280</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>118</red>
             <green>116</green>
             <blue>108</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/balckButton.png);</string>
       </property>
       <property name="text">
        <string>登陆</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageRegister">
      <widget class="QLineEdit" name="regFirstPwd">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>159</y>
         <width>193</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>91</y>
         <width>72</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>261</y>
         <width>72</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>邮  箱：</string>
       </property>
      </widget>
      <widget class="QToolButton" name="regButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>118</red>
             <green>116</green>
             <blue>108</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/balckButton.png);</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>159</y>
         <width>72</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>密  码：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>227</y>
         <width>72</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>手  机：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="regUser">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>91</y>
         <width>193</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="regNickName">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>125</y>
         <width>193</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="regSurePwd">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>193</y>
         <width>193</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="regTitle">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用户注册</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>193</y>
         <width>90</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>确定密码：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>125</y>
         <width>72</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>昵  称：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="regEmail">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>261</y>
         <width>193</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="regPhone">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>227</y>
         <width>193</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageSettings">
      <widget class="QLabel" name="settingsTitle">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>371</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>服务器地址：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="serverIP">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>192.168.31.138</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>服务器端口：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="serverPort">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>80</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QToolButton" name="setSureButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>260</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>118</red>
             <green>116</green>
             <blue>108</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/balckButton.png);</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
